src/compute/src/logging/initialize.rs

```diff
@@ -173,25 +168,30 @@ impl<A: Allocate + 'static> LoggingContext<'_, A> {
         &self,
         event_queue: EventQueue<CB::Container>,
     ) -> Logger<CB> {
-        let mut logger = BatchLogger::new(event_queue.link, self.interval_ms);
-        Logger::new(self.now, self.start_offset, move |time, data| {
-            logger.publish_batch(time, std::mem::take(data));
-            event_queue.activator.activate();
-        })
+        let mut logger = BatchLogger::<_, _>::new(event_queue.link, self.interval_ms);
```

Comment:
```suggestion
        let mut logger = BatchLogger::new(event_queue.link, self.interval_ms);
```

This doesn't do anything, right?